source-code-manager/src/main/java/org/jboss/da/scm/impl/ScmFacade.java

```diff
@@ -38,9 +44,8 @@ public SCMClonner() {
         scmManager.setScmProvider(SCMType.SVN.toString(), new SvnExeScmProvider());
     }
 
-    public void cloneRepository(SCMType scmType, String scmUrl, String revision, File cloneTo)
+    public void shallowCloneRepository(SCMType scmType, String scmUrl, String revision, File cloneTo)
```

Comment:
Not sure about this rename :smiley_cat: This method is kind of lying since we do a full clone if the shallow clone fails.